source/Conversation.cpp

```diff
@@ -566,30 +568,32 @@ bool Conversation::LoadDestinations(const DataNode &node, const ConditionsStore
 	bool hasCondition = false;
 	for(const DataNode &child : node)
 	{
-		if(child.Size() == 2 && child.Token(0) == "goto" && hasGoto)
+		const string &key = child.Token(0);
+		bool hasValue = child.Size() >= 2;
+		if(key == "goto" && hasValue && hasGoto)
```

Comment:
Previously the size condition here was == 2, not >= 2.

Reply:
I'm aware, but there was no particular reason to expect strictly two values. Everything else expects >= 2 values when it only reads the first two values. We only need to looknfor strictly 2 values if having 3 or more values represents something different.

Reply:
ah right